packages/devtools_app/lib/src/shared/framework/screen_controllers.dart

```diff
@@ -104,5 +104,6 @@ class _LazyController<T extends DevToolsScreenController> {
 
 /// Base class for all DevTools screen controllers.
 abstract class DevToolsScreenController extends DisposableController {
-  void init() {}
+  @override
```

Comment:
Is this declaration even necessary if `init()` is declared in `DisposableController`?

Reply:
Good point. Will remove on the next PR.